src/Compilers/CSharp/Portable/Symbols/Synthesized/RefKindVector.cs

```diff
@@ -85,14 +85,14 @@ public string ToRefKindString()
             builder.Append("{");
 
             int i = 0;
-            foreach (int byRefIndex in this.Words())
```

Comment:
>int [](http://example.com/codeflow?start=21&length=3)

`Words()` is `IEnumerable<ulong>`.

Reply:
🤣 I probably should have noticed that when I moved the code

Reply:
The language/tooling should have guard rails in place to make this harder to do by accident. Unfortunate that there isn't a reasonable place for people to insert an "explicit" cast.